packages/code-connect/components/JumpLinks/JumplinkVertical.figma.tsx

```diff
@@ -0,0 +1,25 @@
+import figma from '@figma/code-connect';
+import { JumpLinksItem } from '@patternfly/react-core';
+
+figma.connect(
+  JumpLinksItem,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=5426-8306',
+  {
+    props: {
+      // string
+      tabText: figma.string('Tab Text'),
+
+      // enum
+      isActive: figma.enum('State', { Selected: true }),
+
+      children: figma.children('*')
+    },
+    example: (props) => (
+      // Documentation for JumpLinks can be found at https://www.patternfly.org/components/jump-links
+      <JumpLinksItem href="#" isActive={props.isActive}>
+        {props.tabText}
+        {props.children}
```

Comment:
I don't see any JumpLinksItem in our docs with any text in them other than the tabText. Do you know what this is intended to show?